arch/arm64/src/common/CMakeLists.txt

```diff
@@ -47,6 +46,10 @@ list(APPEND SRCS arm64_syscall.c)
 # Use common heap allocation for now (may need to be customized later)
 list(APPEND SRCS arm64_allocateheap.c)
 
+if(not CONFIG_ARCH_IDLE_CUSTOM)
```

Comment:
Have you tested your commits internally? Please run all tests in vela before submitting to community.

Reply:
Fixed